launchdarkly/metrics_helper.go

```diff
@@ -151,6 +151,12 @@ func baseMetricSchema(isDataSource bool) map[string]*schema.Schema {
 			Computed:    true,
 			Description: "Version of the metric",
 		},
+		ARCHIVED: {
+			Type:        schema.TypeBool,
+			Optional:    !isDataSource,
+			Computed:    true,
```

Comment:
This field is not computed from the user's point of view - it is something they can modify. I think this should be: 
```suggestion
			Computed:    isDataSource,
```